modules/packages/RecordParser.chpl

```diff
@@ -121,11 +121,11 @@ class RecordReader {
 
   /* Create a RecordReader to match an auto-generated regular expression
      for a record created by the :proc:`createRegexp` routine.
-     
+
      :arg t: the record type to read
      :arg myReader: the channel to read from
    */
-  proc RecordReader(type t, myReader) {
+  proc RecordReader(type t, myReader) throws {
```

Comment:
Have we done a `throws` on a constructor?

Reply:
No, but it seems to pass testing. I plan to turn it in to try! if anything goes wrong.